src/layouts/ReviewRequest/components/ApprovedModal/ApprovedModal.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { useDisclosure } from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+import { ComponentMeta, Story } from "@storybook/react"
+
+import { ApprovedModal } from "./ApprovedModal"
+
+const modalMeta = {
+  title: "Components/ReviewRequest/Request Approved Modal",
+  component: ApprovedModal,
+} as ComponentMeta<typeof ApprovedModal>
+
+const Template: Story<never> = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure({ defaultIsOpen: true })
+  return (
+    <>
+      <Button onClick={onOpen}>Open Modal</Button>
+      <ApprovedModal isOpen={isOpen} onClose={onClose}>
+        <Button colorScheme="danger">Click me</Button>
```

Comment:
Minor nit but can this be removed? It was a bit confusing why this was here (same applies for the other stories).

Reply:
to clarify, you meant the entire button right?

Reply:
Yup the entire danger button with "Click me".

Reply:
actually cos this is the playground so i wanted to show the open/close functionality - i could add a `Open Modal` story?

Reply:
clarified offline - modal wasn't showing